### DISCUSSION TITLE
Custom Outputs

### DISCUSSION THREAD

**Discussion by B0BSmiths** [CounterpartyXCP/CIPs#133] [General]
CIP: XXX
Title: Custom Outputs
Author: B0BSmith
Discussions-To: ??
Status: Draft
Type: ??
Created: 2024-1-12

Abstract

Add the option to have custom outputs in a tx

Rationale

Counterparty has "MPMA" Multi Party Multi Asset send and it does this using 2 chained txs and some some clever p2sh, but the second tx results in a tx with no change output so can not be CPFP.  This means they can get stuck in the memepool for a long long time and it is not possible to bump the fee on them 

Copyright
This document is placed in the public domain.

**Comment by jotapea** [comment]
Related: https://github.com/CounterpartyXCP/counterparty-lib/issues/1214
